kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-integ-tests-sample/src/main/java/org/kie/server/springboot/samples/IntegrationTestsWebSecurityConfig.java

```diff
@@ -16,23 +16,33 @@
 
 package org.kie.server.springboot.samples;
 
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.builders.WebSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.userdetails.User;
 import org.springframework.security.crypto.factory.PasswordEncoderFactories;
 import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.provisioning.UserDetailsManager;
 import org.springframework.security.web.firewall.HttpFirewall;
 import org.springframework.security.web.firewall.StrictHttpFirewall;
 
 @Configuration("kieServerSecurity")
 @EnableWebSecurity
 public class IntegrationTestsWebSecurityConfig extends WebSecurityConfigurerAdapter {
 
+    private static final String USER_PASSWORD = "usetheforce123@";
+    private static final String KIE_SERVER_ROLE = "kie-server";
+    private static final String GUEST_ROLE = "guest";
+    private static final String ADMIN_ROLE = "Administrators";
+    private static final String ENGINEERING_ROLE = "engineering";
+    private static final String HR_ROLE = "HR";
+    private static final String IT_ROLE = "IT";
+    private static final String ACCOUNTING_ROLE = "HR";
```

Comment:
maybe you should add here some sonar config to not signal this is security hot spot problem as you hard coded password ;)

Reply:
thank you @mareknovotny - I just added some exclusion rules specifically for sonar but don't see the sonar cloud report to be re-triggered. Is there any way to force it?